scripts/install

```diff
@@ -15,12 +15,18 @@ source /usr/share/yunohost/helpers
 
 random_key=$(ynh_string_random --length=32)
 email=$(ynh_user_get_info --username=$admin --key="mail")
+fpm_footprint="low"
+fpm_free_footprint=0
+fpm_usage="low"
```

Comment:
I wonder if we should worry about these settings for Monica.
Also, if you think we should keep it, the config panel is disabled. We should remove the `.example` at that end.

Reply:
Oh no, I don't think we need it!
I only tried to make monica installable again, we can indeed remove this

Reply:
Okay I’ve tried but I don’t know what to remove haha, I broke everything :p